docs/rule/no-passed-in-event-handlers.md

```diff
@@ -44,6 +44,24 @@ This rule **allows** the following:
 {{foo onClick=this.handleClick}}
 ```
 
+## Configuration
+
+- boolean - `true` to enable / `false` to disable
+
+  - object -- An object with the following keys:
+
+    - `ignore` -- An array of event handler names to ignore. Event handler names should exclude the @ when specifying those intended for named arguments. This rule will ensure both non-named arguments and named arguments are both ignored appropriately.
+
+      eg.
+
+      Given the following configuration:
+
+      ```json
+      { "ignore": [click] }
+      ```
+
+      Both `click` and `@click` will be ignore as violations of this rule.
```

Comment:
We should have them provide the named argument name assuming angle bracket invocation, and then we'd internally handle `@click` when inside angle invocation (because `@click` in angle invocation is the same thing as `click` in mustaches). I think this is what the rule already does, I just want to make sure that the docs make that clear.

Reply:
Are you suggesting that we have them provide the handler in named arg syntax, eg. `@click`?

Reply:
OK fixed, based on our offline discussion.